editor/editor_undo_redo_manager.cpp

```diff
@@ -382,11 +382,15 @@ void EditorUndoRedoManager::set_history_as_unsaved(int p_id) {
 
 bool EditorUndoRedoManager::is_history_unsaved(int p_id) {
 	History &history = get_or_create_history(p_id);
+	if (history.saved_version == -1) {
```

Comment:
It's `uint64_t` so that triggers warnings.

Reply:
Why this doesn't trigger warning? 🙃
https://github.com/godotengine/godot/blob/b7b3a1a78ad07591fa64e471ef5d42c1166e33e7/editor/editor_undo_redo_manager.cpp#L380

Reply:
No idea :(
Seems like they have warnings for `==` but not `=`.